components/combobox/stories/combobox.mdx

```diff
@@ -0,0 +1,89 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
+
+import * as ComboboxStories from './combobox.stories';
+
+<Meta of={ComboboxStories} title="Docs" />
```

Comment:
No worries! This attaches this MDX file to the component, in this case Combobox. MDX files are considered "unattached documentation," so if you want them to live somewhere specific this is how you do that 😄 [Docs are here](https://storybook.js.org/docs/api/doc-block-meta) in case you want more detail.

Reply:
cool, thanks!